### ISSUE TITLE
Final loss when you train your model

### ISSUE THREAD

**Issue by dh562231640** [yihui-he/softer-NMS#15]
Hi I am trying to train a SSD using xxyy and KL loss. May I ask the final loss when you train your model using xxyy and KL loss? It seems location_loss can be a negative number, is that normal?

**Comment by yihui-he** [comment]
The loss can be a negative number. BTW, please have a look at our CVPR version code: https://github.com/yihui-he/KL-Loss
The performance is improved.

**Comment by yihui-he** [comment]
have a look at YOLOv3 + KL-loss https://github.com/yihui-he/KL-Loss#pytorch-re-implementations
